react-front-end/.npmrc

```
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

alternatively

```suggestion
force=true
```

could also work, neither is ideal, as these should be patched upstream